package.json

```diff
@@ -57,6 +57,7 @@
     "hyphenate-style-name": "^1.0.4",
     "immer": "^9.0.12",
     "immerhin": "^0.1.2",
+    "lodash": "^4.17.21",
```

Comment:
I have been adding specific functions only so that we have a kind of control over what we want to use from lodash, otherwise its easy to start using the entire lodash in places where we don't need it ... at least that's the thinking

Reply:
I noticed that and I think it's a maintenance nightmare. Instead, I think we should use specific imports or (better) use an optimization like this one: https://github.com/lodash/babel-plugin-lodash

Reply:
does it work with esbuild?

Reply:
I feel like generally you don't need majority of lodash nowadays, a lot of it is just easier to understand with modern js

Reply:
importing specific packages gives you a thight control over things that you want to be used from lodash. Sometimes you want to replace a thing that lodash does with a different implementation, so having lodash as a dependency is kindof a box once opened hard to close, contributors may start using anything from it and you would have to tell them to just write a simple [].map or something

Reply:
but again if you really really insist, we can drop this idea and start using lodash, then wait until what I am describing becomes a problem and fix it by providing our own utils package that exports lodash functions where needed

Reply:
I think an eslint rule could help to enforce this. We can explore other options with better tree shaking support but lodash is the most popular choice so I'd stick with it for the sake of contributors.

Reply:
We can do a follow up. I'll install a specific package for now.